javascript/javascript1/week3/js/removeTheItem.js

```diff
@@ -0,0 +1,16 @@
+console.log("ITEM ARRAY REMOVAL\n");
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
```

Comment:
', ' is not needed in last index of array or obj

Reply:
I copied the array from the homework.md file. I will keep an eye next time.

I completely agree with your comments. I will improve my code.